healthcheck/healthcheck_test.go

```diff
@@ -2,95 +2,24 @@ package healthcheck
 
 import (
 	"context"
-	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/containous/traefik/testhelpers"
 	"github.com/vulcand/oxy/roundrobin"
 )
 
 const healthCheckInterval = 100 * time.Millisecond
 
-type testLoadBalancer struct {
```

Comment:
Any particular reason why you moved this to the bottom? Does any convention exist that defines the order?